python_scripts/cross_validation_sol_01.py

```diff
@@ -0,0 +1,185 @@
+# ---
+# jupyter:
+#   kernelspec:
+#     display_name: Python 3
+#     name: python3
+# ---
+
+# %% [markdown]
+# # 📃 Solution for Exercise M2.01
+#
+# The aim of this exercise is to make the following experiments:
```

Comment:
General question: is there a way to embed this part, instead of copy-pasting from the exercise document?

Reply:
@carschno you mean that there is a one-on-one duplication from the exercise file right? Probably there would be a way to do that, but clearly the original authors of the course did not think of a way. But you are right it doesn't make much sense and I much prefer the Carpentries workbench here. Anyway, I think it is less effort to keep it like this, if we add our own 'custom' exercises we can of course in the solution part only put the solution :)

Reply:
That is what I meant indeed. But if the answer is 'no' for now, that is how it is.